eva_submission/eload_ingestion.py

```diff
@@ -66,13 +67,23 @@ def ingest(
         clustering_performed_on_assembly = None
         self.eload_cfg.set(self.config_section, 'ingestion_date', value=self.now)
         self.project_dir = self.setup_project_dir()
+
+        if not tasks:
+            tasks = copy(self.all_tasks)
+            tasks.remove('archive_only')
```

Comment:
Should we change the [command line default](https://github.com/EBIvariation/eva-submission/blob/master/bin/ingest_submission.py#L30) as well?